app/views/jobs/show.html.erb

```diff
@@ -1,38 +1,95 @@
-<h1> <%= @job.job_title %>  (job owned by <%= @job.company.user.email%> )</h1>
-
-<p><%= @job.job_description %> </p>
-
-<p>Company: <%= @job.company.name %></p>
-
-<!-- Apply button only showing to developers -->
-<% if @developer_application %>
-  <p>You have already applied to this job. View your <%= link_to 'application', application_path(@developer_application) %></p>
-<% else %>
-  <% if current_user && current_user.developer %>
-    <%= link_to 'Apply now', new_job_application_path(@job) %>
-  <% elsif !current_user %>
-    <p>Please <%= link_to "login", new_user_session_path %> or create an account to apply for this job</p>
-  <% end %>
-<% end %>
-
-<br>
-
-<!-- pundit needs to be set up here -->
-<% if policy(@job).edit? %>
-  <%= link_to 'edit job posting', edit_job_path(@job) %>
-<% end %>
-<br>
-<% if policy(@job).destroy? %>
-  <%= link_to 'delete job posting', job_path(@job), method: :delete %>
-<% end %>
-<!-- only showing application questions and edit/delete if the company owns the jobs posting -->
-<%# if current_user && current_user == @job.company.user %>
-<% if policy(@job).edit? %>
-  <h3>Application questions</h3>
-  <%= link_to 'Add new question', new_job_question_path(@job) %>
-  <ul>
-    <% @questions.each do |question| %>
-      <li> <%= question.content %> <%= link_to 'edit', edit_question_path(question) %> <%= link_to 'delete', question_path(question), method: :delete %></li>
+<div class="container my-5">
+  <div class="title-container">
+    <div class="title-container-left">
+      <h1>Position: <%= @job.job_title %></h1>
+
+      <% if policy(@job).edit? %>
+        <%= link_to 'edit job', edit_job_path(@job) %>
+      <% end %>
+      <br>
+      <% if policy(@job).destroy? %>
+        <%= link_to 'delete job', job_path(@job), method: :delete %>
+      <% end %>
+
+    </div>
+    <!-- Apply now button -->
+    <% if @developer_application %>
+      <p>You have already applied to this job. View your <%= link_to 'application', application_path(@developer_application) %></p>
+    <% else %>
+      <% if current_user && current_user.developer %>
+        <%= link_to 'Apply now', new_job_application_path(@job), class: "btn btn-flat message mt-0" %>
+      <% elsif !current_user %>
+        <p>Please <%= link_to "login", new_user_session_path %> or create an account to apply for this job</p>
+      <% end %>
+    <% end %>
+  </div>
+
+  <!-- container for quick facts -->
+  <div class="job-cards-container">
+    <div class="job-cards">
+      <p>Start date:</p>
+      <p><%= @job.start_date %> </p>
+    </div>
+    <div class="job-cards">
+      <p>Duration:</p>
+      <p><%= @job.duration %> </p>
+    </div>
+    <div class="job-cards">
+      <p>Location:</p>
+      <p><%= @job.location %> </p>
+    </div>
+    <div class="job-cards">
+      <p>Remote:</p>
+      <p><%= @job.remote %> </p>
```

Comment:
Remote is actually a boolean in the schema. So this need an if statement to make it "yes" or "no"